core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestIntToChar.kt

```diff
@@ -0,0 +1,30 @@
+package typeconversion
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class TestIntToChar {
+    @Test
+    fun `Given an Int, When it's valid number, Then should convert Int to the corresponding Unicode character`() {
+        assertEquals('A',  'A')
```

Comment:
should this actually be comparing some number with `.toChar()`?

Reply:
Yes, this was a typo.